Src/AutoFixtureUnitTest/EnumGeneratorTest.cs

```diff
@@ -103,7 +103,7 @@ public void RequestForFlagEnumTypeReturnsCorrectResult(Type enumType, int reques
             var sut = new EnumGenerator();
             // Exercise system
             var dummyContext = new DelegatingSpecimenContext();
-            var result = Enumerable.Repeat<Func<object>>(() => sut.Create(enumType, dummyContext), requestCount).Select(f => f()).Last();
+            var result = Enumerable.Repeat<Func<object>>(() => sut.Create(enumType, dummyContext), requestCount).Select(f => f()).ToArray().Last();

```

Comment:
Yes, see [this](https://github.com/AutoFixture/AutoFixture/pull/773/commits/764f10899fb8ee25ef20f38c9fc4041122c1a783) commit message.